agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -244,7 +244,10 @@ public String toString() {
 
     public List<InstrumentationKeyOverride> instrumentationKeyOverrides = new ArrayList<>();
 
-    public int exportQueueCapacity = 2048;
+    public int generalExportQueueCapacity = 2048;
+    // metrics get flooded every 60 seconds by default, so need larger queue size to avoid dropping
+    // telemetry (they are much smaller so a larger queue size is ok)
+    public int metricsExportQueueCapacity = 65536;
```

Comment:
32 times more.. how did you come up with this number?

Reply:
how did you find out it got dropped? i wonder if we can utilize statsbeat to catch it.

Reply:
> how did you find out it got dropped?

we log when we drop something

> i wonder if we can utilize statsbeat to catch it.

sure (but not in this PR) 👍